modules/programs/wpaperd.nix

```diff
@@ -1,49 +0,0 @@
-{ config, lib, pkgs, ... }:
-
-with lib;
-
-let
-  cfg = config.programs.wpaperd;
-  tomlFormat = pkgs.formats.toml { };
-in {
-  meta.maintainers = [ hm.maintainers.Avimitin ];
```

Comment:
Should probably check if the previous maintainer is interested in maintaining still.

Reply:
Sorry, didn't see that. Thanks